source/mainnet/smart-contracts/tutorials/nft-minting/setup-dev-env.rst

```diff
@@ -0,0 +1,108 @@
+.. include:: ../../../variables.rst
+.. _setup-dev-env:
+
+=================================
+Setup the development environment
+=================================
+
+First, you need to install “rustup” which installs Rust and Cargo to your computer. To do that, you can use this command.
+
+.. code-block:: console
+
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
This is only linux/mac. Please link to rustup https://rustup.rs/ which will have platform specific instructions.